client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/site-setup-design-picker.tsx

```diff
@@ -335,31 +336,49 @@ const SiteSetupDesignPicker: Step = ( { navigation, flow } ) => {
 			/>
 		);
 
+		let header = (
+			<FormattedHeader
+				id={ 'design-setup-header' }
+				headerText={ designTitle }
+				align={ isMobile ? 'left' : 'center' }
+			/>
+		);
+
+		if ( intent === 'sell' ) {
+			if ( storeType === 'simple' ) {
```

Comment:
I believe the "Multi-product store" title at the top of the mockup image is meant to be the theme name, rendered dynamically.

Reply:
In that case I believe the only change is gonna be the black info button😄, considering that:
1. Should display site preview in the same manner as existing behavior
2. Behavior for the info and "upgrade plan" button will follow in separate issues
3. Ignore "WooCommerce" flair for theme, as we don't have any themes meeting this criteria yet.

Reply:
OK, fair enough. `tweaks` --> `tweak`; for some reason I thought there were more differences to bring in line.